docs/design/traffic-acceleration.md

```diff
@@ -0,0 +1,148 @@
+      # noEncap or hybrid. External-to-Service and Pod-to-external traffic will be supported in the future.
+      enable: true
+```
+
+## Implementation
+
+### Table
+
+Antrea introduces dedicated nftables tables named `antrea` for both IPv4 and IPv6:
+
+```text
+table ip antrea {
+    comment "Rules for Antrea"
+}
+table ip6 antrea {
+    comment "Rules for Antrea"
+}
+```
+
+These tables contain flowtables, sets, and chains used for acceleration logic.
+
+### Peer Pod CIDR Set
+
+Each nftables table maintains a set named `peer-pod-cidr` containing peer Pod CIDRs of remote Nodes.
+
+IPv4 example:
+
+```text
+table ip antrea {
+    set peer-pod-cidr {
+        type ipv4_addr
+        flags interval
+        comment "Set for storing IPv4 peer Pods CIDRs"
+        elements = { 10.244.0.0/24, 10.244.1.0/24 }
+    }
+}
+```
+
+IPv6 example:
+
+```text
+table ip6 antrea {
+    set peer-pod-cidr {
+        type ipv6_addr
+        flags interval
+        comment "Set for storing IPv6 peer Pods CIDRs"
+        elements = { fd00:10:244::/64, fd00:10:244:1::/64 }
+    }
+}
+```
+
+### Flowtable
+
+The flowtable named `fastpath` is bound to both the Antrea gateway interface (`antrea-gw0`) and the Node’s transport
+interface (e.g., `eth0`). This configuration allows packets forwarded between these interfaces to be accelerated.
+
+```text
+table ip antrea {
+    flowtable fastpath {
+        hook ingress priority filter
+        devices = { antrea-gw0, eth0 }
+    }
+}
+```
+
+### Forwarding Chain
+
+The chain named `forward` holds the rules to match the connections which are eligible for flowtable acceleration.
+Currently, only inter-Node Pod-to-Pod traffic in noEncap and hybrid modes is supported.
+
+```text
+table ip antrea {
+    chain forward {
+        comment "Forward chain for storing rules"
+        type filter hook forward priority filter; policy accept;
+
+        iif "antrea-gw0" ip saddr 10.244.2.0/24 oif "eth0" ip daddr @peer-pod-cidr \
+            flow add @fastpath counter packets 0 bytes 0 \
+            comment "Accelerate IPv4 connections: local Pod CIDR to remote Pod CIDRs"
+
+        iif "eth0" ip saddr @peer-pod-cidr oif "antrea-gw0" ip daddr 10.244.2.0/24 \
+            flow add @fastpath counter packets 0 bytes 0 \
+            comment "Accelerate IPv4 connections: remote Pod CIDRs to local Pod CIDR"
+    }
+}
+```
+
+In the future, Antrea will support more traffic types:
+
+- External-to-Service traffic: accelerate external client traffic destined for a Service before it enters the Antrea
+  OVS pipeline on Node host networking, in encap, noEncap, or hybrid modes.
+- Pod-to-external traffic: accelerate Pod egress traffic after it leaves the Antrea OVS pipeline on Node host networking,
+  in encap, noEncap, or hybrid modes.
+
+## Benchmark
+
+The following results are the netperf results of encap, noEncap without acceleration, noEncap with acceleration:
```

Comment:
How about: “netperf tests were conducted to compare performance across three configurations: encap (no acceleration), noEncap (no acceleration), and noEncap (with acceleration).”